tests/testthat/test-other-colorKey-Numeric.R

```diff
@@ -0,0 +1,36 @@
+context("canvasXpress colorKey for Numeric key")
```

Comment:
simplify to match file name: **canvasXpress numeric colorkey**
change filename to: test-other-numeric-colorkey.R

Reply:
renamed the file